src/pages/Dashboard/Markets/SupplyMarket/index.stories.tsx

```diff
@@ -14,14 +14,15 @@ export default {
   subcomponents: { SupplyMarketUi },
   parameters: {
     backgrounds: {
-      default: 'Primary',
+      default: 'White',
```

Comment:
why did you use white here?

Reply:
So that you could see the difference between the background of the story and the component being displayed better